tlsn/tlsn-core/src/substrings/commitment.rs

```diff
@@ -43,13 +42,11 @@ impl SubstringsCommitmentSet {
         }
 
         // grand total in all of the commitments' ranges must be sane
-        let mut total_committed = 0u64;
+        let mut total_committed = 0;
         for commitment in &self.0 {
-            for r in commitment.ranges() {
-                total_committed += (r.end - r.start) as u64;
```

Comment:
earlier `r.end` and `r.start` were u32, so `total_committed` which was u64 could not be overflown. With the change to usize it can be overflown.